src/show/components/Footer/index.jsx

```diff
@@ -1,5 +1,29 @@
+import { BsGithub } from "react-icons/bs"
+import { BsFacebook } from "react-icons/bs"
+import { BsInstagram } from "react-icons/bs"
+import { BsSpeedometer2 } from "react-icons/bs"
```

Comment:
You are importing 4 different items from the same place, you can do this in one line.